### ISSUE TITLE
Глоссарий

### ISSUE THREAD

**Issue by frost-nzcr4** [avalanche123/symfony-docs-ru#17]
Предлагаю существующий http://trac.symfony-project.org/wiki/РусскийГлоссарийSymfony

**Comment by Skorney** [comment]
да, я не против этого глоссария, думаешь стоит включить его как-то в README?

**Comment by frost-nzcr4** [comment]
думаю его следует дополнить и использовать по назначению, можно и в ридми его поместить, а можно как issue оставить

**Comment by Skorney** [comment]
Я думаю что можно создать страничку wiki с глоссарием, и на нее ссылаться с README

**Comment by frost-nzcr4** [comment]
я тоже за

**Comment by stfalcon** [comment]
здесь немного есть http://hudson.su/2011/03/28/symfony2-glossary/

**Comment by frost-nzcr4** [comment]
Дополнение в список кандидатов на правильный перевод или "неперевод" вовсе: Assertion, Crawler.

**Comment by stfalcon** [comment]
как переводить Assetic Filter? например "How to Apply an Assetic Filter to a Specific File Extension". сейчас пишу "Применение Assetic фильтров к определенным расширениям файлов"
